composer.json

```diff
@@ -18,22 +18,11 @@
         "ext-simplexml": "*",
         "guzzle/guzzle": "^2.8|^3.0",
         "psr/log": "^1.0",
-        "symfony/config": "^2.0|^3.0",
+        "symfony/config": "^2.6|^3.0",
```

Comment:
The autoloader with 2.0 doesn't work on circleci. I didn't dig deeper, since it's easier to use 2.6 (the last version that supports PHP 5.3.3).